### DISCUSSION TITLE
Roadmap to Becoming a Developer and getting known.

### DISCUSSION THREAD

**Discussion by siddhant-khisty** [EddieHubCommunity/support#2577] [Show and tell] [answered]
A blog made for new developers for getting started

**Comment by siddhant-khisty** [comment, marked as answer]
https://dev.to/siddhantkhisty/roadmap-to-becoming-a-developer-and-getting-known-24o4

**Reply by Panquesito7** [reply]
Awesome one! Let's share it on Twitter. 🎉

**Comment by Kernix13** [comment]
Do you have a website and are you open to adding a blog post on my website? I'm way behind on articles but it wouldn't only make sense for you if you have a website that you can link to from my site.
